libs/Common/Soap/CurlSoap.php

```diff
@@ -97,15 +131,15 @@ public function getProxy()
     
     /**
      * Envia mensagem ao webservice
-     * 
      * @param string $urlsevice
      * @param string $namespace
      * @param string $header
      * @param string $body
      * @param string $method
+     * @param string $data Variavel passada como referencia para retorna a mensagem enviada
      * @return boolean|string
      */
-    public function send($urlservice, $namespace, $header, $body, $method)
+    public function send($urlservice, $namespace, $header, $body, $method, &$data = '')
```

Comment:
@robmachado lembro que em outro PR já havia comentado a respeito de usar referência, isso torna o código mais lento!

- faz sentido retornar a mensagem enviada?
- Qual a finalidade?

Se for realmente necessário, sugiro que refatore para não usar referência, uma sugestão é adicionar uma propriedade para armazenar a última mensagem enviada e um método para pegar este valor.